src/App.js

```diff
@@ -8,40 +8,64 @@ var Production = loadContract("Production")*/
 import ProductionCreator from "./components/ProductionCreator.js"
 import ProductionList from "./components/ProductionList.js"
 import RegistrySetter from "./components/RegistrySetter.js"
+import RegistryViewer from "./components/RegistryViewer.js"
+import Login from "./components/Login.js"
 import Identicon from "react-blockies"
 
 class App extends Component {
 	constructor(props) {
 		super(props)
-		this.state = {prodList: [], registry: null}
+		this.state = {prodList: [], registry: null, address: null}
 	}
 	addProd(prod) {
-		console.log("addProd: "+prod)
 		this.setState({prodList: this.state.prodList.concat([prod])})
 	}
 	setRegistry(reg) {
 		this.setState({registry: reg})
 	}
+	setAddress(addr) {
+		this.setState({address: addr})
+	}
 	render() {
-		console.log("prodList: "+this.state.prodList)
 		var regStr = this.state.registry ? this.state.registry.address : "none"
+		var addrStr = this.state.address ? this.state.address : "none"
```

Comment:
let's name vars in a understandable way, like no addrStrSmthg or so, no IoT compiled file-size to optimise here ;)